packages/python-neo4j-driver/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+#
+# This PKGBUILD was extracted from AUR.
+# Creator: tuftedocelot tuftedocelot@fastmail.fm<
+# Changes by: petruknisme
+
+_pkgname=neo4j-python-driver
+pkgname=python-neo4j-driver
+pkgver=1.7.6
+pkgrel=1
+pkgdesc='Neo4j Bolt driver for Python'
+arch=('any')
+url='https://github.com/neo4j/neo4j-python-driver'
+license=('Apache')
+depends=('python' 'python-pytz')
+makedepends=('python-setuptools')
+source=("https://github.com/neo4j/$_pkgname/archive/$pkgver.zip")
+sha512sums=('36e5f12ca7f61f1e174914e274b7ba5af82a159bdd9c5830c23a2908da548b872116ad7230dc9a9966922f96f5f7351060fab2b42537938e7c9f020ce6e69b95')
+
+build(){
+  cd $_pkgname-$pkgver
+
+  python setup.py build
+}
+
+package() {
+  cd $_pkgname-$pkgver
+
+  python setup.py install --root="$pkgdir/" --prefix=/usr -01 --skip-build
```

Comment:
Please fix this typo. It's `O1` (optimize) not `01`. Every time you make a change, test with ba-dev.

Reply:
Oh sorry, that was typo. I'm running ba-dev first before submitting to repo.